### PR TITLE
Send New Rent Email

### PR DESCRIPTION
**Pull request by cquiro**
## Summary

- Add sidekiq to Gemfile and bundle install
- Set action_job.queue_adapter to sidekiq
- Set action_mailer.delivary method to :smtp and set configuration to use mailtrap
- Add mailtrap credentials to config/credentials.yml.enc
- Create RentsMailer
- Add slim-rails to Gemfile and bundle install
- Write html email
- Call RentsMailer.send_new_rent_email from the create action of the rents_controller

## Trello Card

https://trello.com/c/JdlxdT35/11-mandar-email-al-usuario-cuando-se-crea-un-rent

## Screenshot

screen shot 2018-09-28 at 4 32 55 pm

### CODE REVIEW

File: config/environments/production.rb [hunk 1]
```diff
@@ -65,6 +65,17 @@
 
   config.action_mailer.perform_caching = false
 
+  config.action_mailer.delivery_method = :smtp
+
+  config.action_mailer.smtp_settings = {
+    :user_name => Rails.application.credentials.mailtrap[:username],
```
Comment (facucompe) [inline config/environments/production.rb]: You can use new sintax

File: config/environments/production.rb [hunk 2]
```diff
@@ -65,6 +65,17 @@
...
+    :password => Rails.application.credentials.mailtrap[:password],
+    :address => 'smtp.mailtrap.io',
```
Comment (facucompe) [inline config/environments/production.rb]: I would add this to the env variables

**Review by facucompe — CHANGES_REQUESTED** [review]
Just those small things

**Review by facucompe — APPROVED** [review]

File: app/mailers/application_mailer.rb [hunk 3]
```diff
@@ -1,4 +1,4 @@
 class ApplicationMailer < ActionMailer::Base
-  default from: 'from@example.com'
+  default from: 'info@wbooks.com'
```
Comment (lmasello) [inline app/mailers/application_mailer.rb:2]: There is no need to change it, but just for your information, this is preferable to be taken from an environment variable :)

**Review by lmasello — APPROVED** [review]
Well done!

**Pull request merged**